### DISCUSSION TITLE
Wazuh behind Apache2 with ModSecurity

### DISCUSSION THREAD

**Discussion by arturfromtheblock** [wazuh/wazuh#26456] [General]
Hey there.

First of all thanks for that incredible tool that you guys are developing. Just had in the beginning a lot of issues in the log files, but not it's almost there where I want to have it.
I wonder if anyone here had managed to run Wazuh with the mod security installed for Apache2?
So Wazuh is running, I can access the Dashboard with my custom subdomain, the only issue is, that the mod security is blocking a lot of wazuh request (mostly api request). 

Can somebody share a how-to because I couldn't find anything on the internet related to that? I didn't want to disabled rules to archive that to keep Wazuh safe.

Regards
![screen 2024-10-21 um 17 40 07](https://github.com/user-attachments/assets/9e4bd463-6875-448b-825d-75a8bcedab3b)

**Comment by SirStephanikus** [comment]
You have to open the requests by yourself, after you did a stringent monitoring of your logs in transparent mode of mod_security ... this is the essence of every WAF. 

My advice, do the mod_security course (will take around 1-2 weeks in full-time) and read the mod_security book to get an understanding how the system works.

Just relying on something you found in google or asking for a how-to does not help.
